sql/rowexec/dml.go

```diff
@@ -85,9 +85,6 @@ func (b *BaseBuilder) buildInsertInto(ctx *sql.Context, ii *plan.InsertInto, row
 func (b *BaseBuilder) buildDeleteFrom(ctx *sql.Context, n *plan.DeleteFrom, row sql.Row) (sql.RowIter, error) {
 	// If an empty table is passed in (potentially from a bad filter) return an empty row iter.
 	// Note: emptyTable could also implement sql.DetetableTable
-	if _, ok := n.Child.(*plan.EmptyTable); ok {
```

Comment:
should we delete the comment?

Reply:
will delete comment.

if `n.Child` is an `Exchange` node it fails the EmptyTable check here, an alternative would be to use transform.Inspect to look for EmptyTable maybe?

Reply:
The current pattern seems fine, folding x->EmptyTable into EmptyTable is analyzer responsibility not executions